validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/KeystoresValidatorKeyProvider.java

```diff
@@ -82,7 +84,7 @@ private String loadPassword(final Path passwordFile) {
           format(
               "Unexpected IO error while reading keystore password file [%s]: %s",
               passwordFile, e.getMessage());
-      STATUS_LOG.log(Level.FATAL, errorMessage);
+      LOG.fatal(errorMessage);
```

Comment:
Fatal error probably should go to stdout.  The user would want to know that we won't be running their validator so they will be losing money.

Reply:
The error message makes its way into the stdout, by virtue of being converted to a `RuntimeException` subclass and caught by the `BeaconNode.start()` `catch` block.

Perhaps this log line is not required here (as it causes duplicate logging), without any additional information?

Reply:
Yes, we should either catch and log, *or* rethrow but not both (unless we can provide some extra unique information at this level that will be lost higher up).

Reply:
Thinking about this after finishing my first coffee of the day ...I should be deleting this duplicate logging,. Even if the message needs to be displayed in two seperate places that is probably best achieved with logging configuration, rather then programatically,

Reply:
Yep, all messages that go to stdout should also go to the log file so you have a complete record in the log file.